src/edu/csus/ecs/pc2/core/execute/ExecuteUtilities.java

```diff
@@ -767,4 +767,19 @@ public static void writeFileContents(String filename, ArrayList<String> dataline
         writer.close();
         writer = null;
     }
+    
+    
+    /**
+     * Cast a CheckedException as an unchecked one.
```

Comment:
I think the method description is a bit misleading; it says that what it does is "Cast" an exception -- but it does more than that; it also _throws_ the (cast) exception.